src/Test/test_linear.jl

```diff
@@ -4064,6 +4064,47 @@ end
 
 version_added(::typeof(test_linear_open_intervals)) = v"1.7.0"
 
+"""
+    test_linear_variable_open_intervals(
+        model::MOI.ModelLike,
+        config::Config{T},
+    ) where {T}
+
+Test that the solver supports open-intervals like `(-Inf, u]`, `[l, Inf)`, and
+`(-Inf, Inf)`.
+"""
+function test_linear_variable_open_intervals(
+    model::MOI.ModelLike,
+    config::Config{T},
+) where {T}
+    @requires _supports(config, MOI.optimize!)
+    @requires MOI.supports_constraint(model, MOI.VariableIndex, MOI.Interval{T})
```

Comment:
Wasn't there an issue with this and solvers which bridged via SplitInterval? I forget where we left things with that.

Reply:
We updated splitinterval to not pass on the infinite bounds.